modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/styles/_data_set.scss

```diff
@@ -39,3 +39,15 @@
 		}
 	}
 }
+
+.data-set-wrapper {
+	border: 2px solid transparent;
+}
+
+.data-set-wrapper.drop-target {
+	& *:not(.btn-primary):not(.lexicon-icon-plus):not(.lexicon-icon-plus use) {
+		@include common-drop-target;
+	}
+	@include common-drop-target;
+	border: 2px solid $info !important;
```

Comment:
Do we need `!important` here? Generally it is good to avoid it, unless it's it an override of inline styles, it something crazy like we're doing with `common-drop-target`.